crates/viewer/re_blueprint_tree/src/blueprint_tree.rs

```diff
@@ -118,20 +118,27 @@ impl BlueprintTree {
         self.candidate_drop_parent_container_id = self.next_candidate_drop_parent_container_id;
         self.next_candidate_drop_parent_container_id = None;
 
-        let filter_matcher = self.filter_state.filter();
+        let blueprint_tree_data =
+            std::hint::black_box(BlueprintTreeData::from_blueprint_and_filter(
```

Comment:
Using `std::hint::black_box` outside of a benchmark is very surprising. Why is it here?